docs/griptape-tools/official-tools/google-gmail-client.md

```diff
@@ -26,42 +26,58 @@ gmail_tool = GoogleGmailClient(
 
 # Set up an agent using the GoogleGmailClient tool
 agent = Agent(
-    tools=[gmail_tool]
+    tools=[FileManager(), gmail_tool]
```

Comment:
Let's move `FileManager` initialization next to `gmail_tool` so that the Tools are consistently initialized.

Reply:
ok!